nservicebus/messaging/third-party-integration.md

```diff
@@ -1,43 +1,37 @@
 ---
 title: Third Party Integration
-summary: Messaging scenarios with non-NSB systems
+summary: Receiving messages from external systems
+component: Core
 tags:
 - Monitoring
 - Third Party Integration
 - ServiceInsight
-- ServicePulse
 related:
 - nservicebus/messaging/headers
+reviewed: 2017-01-18
 ---
 
-When an NServiceBus endpoint receives messages from external systems (such as BizTalk, TIBCO, etc) the message itself might not contain regular all information that NServiceBus provides. That information is necessary for proper serialization of the message, as well as better monitoring and troubleshooting experience.
-
-To get the best use of the tooling ensure that the following information is included in the message sent by the third party endpoints.
+NServiceBus endpints can receive messages from external systems (such as BizTalk, TIBCO, etc). To ensure those messages can be handled correctly by NServiceBus, additional information might be required which are otherwise provided by NServiceBus automatically.
```

Comment:
Also add another sentence to further state that, when receiving messages from these external systems, it is therefore important to ensure that the user adds this extra information. The next question then becomes how?